src/applications/disability-benefits/all-claims/pages/hospitalizationHistory.js

```diff
@@ -68,21 +57,7 @@ export const schema = {
     unemployability: {
       type: 'object',
       properties: {
-        hospitalProvidedCare: {
-          type: 'array',
-          items: {
-            type: 'object',
-            properties: {
-              name: hospitalProvidedCare.items.properties.name,
-              'view:hospitalAddressTitle': {
-                type: 'object',
-                properties: {},
-              },
-              address: addressSchema,
-              dates: hospitalProvidedCare.items.properties.dates,
-            },
-          },
-        },
+        hospitalProvidedCare,
```

Comment:
Look at that! Use the schema provided and it's so clean. :smile:

Reply:
💯